### ISSUE TITLE
Add an option to reorder the links

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/external#70]
Extracted from https://github.com/nextcloud/external/issues/69 by @Dubidubiduu

> Then I would appreciate a way to change the order of my external sites which is not possible. But there is an App to change the order of menu items ("App Order"), so this is not a problem, but a small hint in the description of this app would be a nice thing.

**Comment by sunjam** [comment]
Fyi, I’m able to reorder external links with no problems in latest Nextcloud 13 via [Apporder](https://apps.nextcloud.com/apps/apporder)

**Comment by Dubidubiduu** [comment]
Actually only a few external links are shown in the "app order"-app. Maybe just the first ones. Maybe this is because of the limitation of items in the original menu?!

But I don't have the limitation of items in my menu, since I use "sidebar menu-app", gonna write to the "app-order" app about this.

**Comment by Luticus** [comment]
@Dubidubiduu Well the app-order app does work, as long as your external sites are all in the app menu.  In that case it's fine, and I've actually been doing that for a while now.  My current issue is that I've places some external sites in the settings drop down because they are more admin related, and I've used the external sites group restrictions to block the sites from being shown to normal users.  In that case there is no way at all to reorder them without deleting and adding them again, as far as I can tell.

**Comment by Dubidubiduu** [comment]
@Luticus Yeah but for this I added an issue in the app-order-app https://github.com/juliushaertl/apporder/issues/87

**Comment by nunomdmgoncalves** [comment]
Defining an app order based on groups or and admin could set a specific app order on a user would be great, that way a user did not get an access denied if it has not permissions to a specific app. Another way is app order jump to the next permitted app in the specified order.
that would be great !

**Comment by Luticus** [comment]
@Dubidubiduu I think there's some confusion here as I don't see how that app order issue will address what I'm a talking about.  

Basically if the app/icon/menu item is in the apps list: 
![apporderbar](https://user-images.githubusercontent.com/7807109/81203033-6b38b900-8f95-11ea-998b-4c6e5bb534c1.png)
Then I agree, the app order app helps a lot.

However if the items are in this menu:
![appordersettingmenu](https://user-images.githubusercontent.com/7807109/81203206-ae932780-8f95-11ea-96e2-b820b419a37a.png)
Then the app order app has no bearing on their position and they cannot easily be rearranged.

**Comment by nroach44** [comment]
For anyone curious, this can be achieved through some database wrangling. The following is how I did it with Postgres:

Enter the postgres command line, find your DB, and use it:

```
sudo -i -u postgres
psql
\l (list databases)
\c {{ YOUR NEXTCLOUD DATABASE NAME }}
```

Grab the config:

```
select configvalue from oc_appconfig WHERE appid = 'external' AND configkey = 'sites';
```

Pipe this into your favourite JSON viewer / expander / indent tool.

Move the order of them around, and then re-number the IDs (both in the record and at the start of it).

Remove all new lines from the updated JSON to put it back in the same format as you found it.

Apply the updated config:

```
UPDATE oc_appconfig SET  configvalue = {{ YOUR MINIFIED JSON }} WHERE appid = 'external' AND configkey = 'sites';
```
